corehq/apps/app_manager/views/app_summary.py

```diff
@@ -396,7 +396,7 @@ def get(self, request, domain, app_id):
         case_metadata = self.app.get_case_metadata()
         language = request.GET.get('lang', 'en')
 
-        headers = [('All Case Properties', ('case_type', 'case_property'))]
+        headers = [('All Case Properties', ('case_type', 'case_property', 'description'))]
```

Comment:
did you want translate tags just for the phrase, or all the underscore properties as well